ddl/rollingback.go

```diff
@@ -311,7 +311,11 @@ func convertJob2RollbackJob(w *worker, d *ddlCtx, t *meta.Meta, job *model.Job)
 			logutil.Logger(w.logCtx).Info("[ddl] the DDL job is cancelled normally", zap.String("job", job.String()), zap.Error(err))
 		}
 
-		job.Error = toTError(err)
+		if job.Error == nil {
+			job.Error = toTError(err)
+		} else {
+			job.Error = toTError(job.Error.GenWithStack("[%s] %s", toTError(err).ToSQLError().Message, job.Error.ToSQLError().Message))
```

Comment:
"current error msg:  %s, original error msg: %s"

Besides, if we encounter an error in the function of "rollingbackXxxx", we may get the error `current error msg:  err1, original error msg: err0`. Then we rolling back the DDL job again. In the end, we may get the error like `current error msg:  err2, original error msg: current error msg:  err1, original error msg: err0`. 
Perhaps we can confirm that the initial error message is not overwritten at the time of rolling back, and that this error format is handled only when the rollingback ends.

Reply:
Thanks, I had modified it. PTAL @zimulala